pallets/runtime/mainnet/src/runtime.rs

```diff
@@ -71,6 +71,9 @@ parameter_types! {
     pub const EpochDuration: u64 = EPOCH_DURATION_IN_BLOCKS as u64;
     pub const ExpectedBlockTime: Moment = MILLISECS_PER_BLOCK;
 
+    // Base:
+    pub const MaxLen: u32 = 256;
```

Comment:
Let's bump this to 2048
```suggestion
    pub const MaxLen: u32 = 2048;
```